### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by theitush**

### CODE REVIEW

**Comment by theitush** [comment]
Still WIP.

**Comment by theitush** [comment]
Basic tests for Project_Runner!
I can finally stop breaking things..!

File: Python_pipeline/tests/testProjectRunner.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    if filename.find("tau.ac.il") == -1:
+        return False
+    else:
+        return True
+
+
+class TestProjectRunner(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        #TODO?: purge output dir if it's overcrowded..?
```
Comment (daniellemiller) [inline Python_pipeline/tests/testProjectRunner.py:34]: sounds logical - maybe consider running tests under "no backup"?

Comment (theitush) [reply Python_pipeline/tests/testProjectRunner.py:34]: niiiice

**Comment by theitush** [comment]
> Awesomeness.
> The tests are running after the bash command ends?

You basically run: `python Python_pipeline/tests/testProjectRunner.py` what it does is run Project_Runner on a small subset of sample data and checks that all the right files are created and that you have an AggregatedSummary.csv which is identical to an example file.

**Pull request merged**